logger/Logger.h

```diff
@@ -313,6 +317,15 @@ class Logger
     static bool fIsDestructed;
     static struct DestructionHelper { ~DestructionHelper() { Logger::fIsDestructed = true; }} fDestructionHelper;
 
+    static bool constexpr SuppressSeverity(Severity sev)
+    {
+#ifdef FAIR_MIN_SEVERITY
+        return sev < Severity::FAIR_MIN_SEVERITY;
```

Comment:
Codacy found an issue: [Skipping configuration 'FAIR_MIN_SEVERITY;NDEBUG' since the value of 'FAIR_MIN_SEVERITY' is unknown. Use -D if you want to check it. You can use -U to skip it explicitly.](https://app.codacy.com/manual/FairRootGroup/FairLogger/pullRequest?prid=5713891)